src/main/java/com/tutrit/bean/User.java

```diff
@@ -0,0 +1,8 @@
+package com.tutrit.bean;
+
+public record User(
```

Comment:
model class should be at `core` module, 
please take another look at https://github.com/max-java/quickstart-multimodule-maven

Reply:
I've understood you. I'll change it